### PR TITLE
Medalla guide updates + troubleshooting page

### PR DESCRIPTION
**Pull request by sachayves**
- explain how to attach multiple validators to the same beacon node
- add dedicated medalla troubleshooting page
- remove duplicate information around keys

### CODE REVIEW

File: docs/the_nimbus_book/src/medalla-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Troubleshooting Medalla
+
+As it stands, we are continuously making improvements to both stability and memory usage. So please make sure you keep your client up to date! This means restarting your node and updating your software regularly (we recommend doing this at least once a day). To update and restart, run `git pull`, `make update`, followed by `make medalla`:
```
Comment (mratsim) [inline docs/the_nimbus_book/src/medalla-troubleshooting.md:3]: For a long time, master was lagging behind devel. Maibe we should have a paragraph saying "While helping you, we might ask you to run the latest `devel` in that case use `git checkout devel`". Note: this branch is our development branch and may be in an unstable state.

File: docs/the_nimbus_book/src/medalla-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+>**Note:** rest assured that when you restart the beacon node, the software will resume from where it left off, using the validator keys you have already imported.
+
+
+### Starting over
+
+The directory that stores the blockchain data of the testnet is `build/data/shared_medalla_0` (if you're connecting to another testnet, replace `medalla` with that testnet's name). Delete this folder to start over (for example, if you started building medalla with the wrong private keys).
+
+### Syncing
+If you’re experiencing sync problems, or have been running an old version of medalla, we recommend running `make clean-medalla` to restart your sync (make sure you’ve updated to the latest `devel` branch first though).
```
Comment (mratsim) [inline docs/the_nimbus_book/src/medalla-troubleshooting.md:22]: The `devel` branch needs to be introduced.

**Review by mratsim — COMMENTED** [review]
Looks good.

Note, the EF wants client teams to explain how to migrate to and from other clients so the key management would have to be expanded (at a later time) with how to import/export keys to/from Prysm/Lighthous/Teku/Lodestar

**Review by arnetheduck — APPROVED** [review]

File: docs/the_nimbus_book/src/medalla-troubleshooting.md (hunk 2 above)
Comment (sachayves) [reply docs/the_nimbus_book/src/medalla-troubleshooting.md:29]: done

File: docs/the_nimbus_book/src/medalla-troubleshooting.md (hunk 1 above)
Comment (sachayves) [reply docs/the_nimbus_book/src/medalla-troubleshooting.md]: as discussed, keeping `devel` as the recommended branch to build from (for now). added a disclaimer and recommendation to fall back to master if `devel` is proving too unstable

**Pull request merged**